client/my-sites/checkout/cart/cart-messages.tsx

```diff
@@ -113,7 +114,7 @@ function getInvalidMultisitePurchaseErrorMessage( {
 		<>
 			{ message }&nbsp;
 			<a
-				href={ JETPACK_SUPPORT + 'backup/#does-jetpack-backup-support-multisite' }
+				href={ localizeUrl( JETPACK_SUPPORT ) + 'backup/#does-jetpack-backup-support-multisite' }
```

Comment:
Nit: I think the full concatenated URL string should be wrapped in **localizeUrl**, i.e. `localizeUrl( JETPACK_SUPPORT + 'backup/#does-jetpack-backup-support-multisite' )`. Though, the result should be the same.

Reply:
Good point